easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb

```diff
@@ -223,7 +223,7 @@ exts_list = [
         'checksums': ['9cd9313954f6ecd82480d373f6c5371ca84ab33e3f5c39d972d35cfcf1096846'],
     }),
     ('MASS', '7.3-54', {
-        'checksums': ['b800ccd5b5c2709b1559cf5eab126e4935c4f8826cf7891253432bb6a056e821'],
+        'checksums': ['eb644c0e94b447c46387aa22436ef5a43192960ee9cfd0df2940f4a4116179ae'],
```

Comment:
@SIB-software Rather than hard replacing the checksum, we should list both as valid checksums using a 2-tuple, since only publication date in the `DESCRIPTION` file in these source tarballs has changed:

```python
    ('MASS', '7.3-54', {
        'checksums': [('b800ccd5b5c2709b1559cf5eab126e4935c4f8826cf7891253432bb6a056e821',
                       'eb644c0e94b447c46387aa22436ef5a43192960ee9cfd0df2940f4a4116179ae')],
```

This way people with an already downloaded source tarball won't run into trouble.

Reply:
Perfect thanks. Yes I thought about that problem of people who already have the source but I didn't know it's possible to provide 2 checksum values for a given source.

I think the same issue also affects `R-4.1.2-foss-2021b.eb`. I will double check and make a PR if needed.